source/developers-guide/debugging/index.md

```diff
@@ -60,42 +71,63 @@ xdebug.remote_port=9000
+Equivalent extensions exist for other browsers.
+Should your browser not have or support these extensions, you might still use Xdebug by appending a specific query argument to your URL.
+PhpStorm will do this automatically for you if you configure the debug environment, but this feature is not exclusive to this IDE.
+Refer to Xdebug and your IDE's documentation for more info on this feature.  
 
 This might be very useful, as Xdebug might reduce the overall performance of your request.
 
-If you are debugging a CLI command, you can also use Xdebug. Use `export XDEBUG_CONFIG="idekey=PHPSTORM"` prior to running your PHP CLI command and make sure that PhpStorm is listening for Xdebug connections. Again, this feature is not exclusive to PhpStorm, and might be supported in other IDEs.
+If you are debugging a CLI command, you can also use Xdebug.
+Use `export XDEBUG_CONFIG="idekey=PHPSTORM"` prior to running your PHP CLI command and make sure that PhpStorm is listening for Xdebug connections.
+Again, this feature is not exclusive to PhpStorm, and might be supported in other IDEs.
 
 ### Xdebug and IonCube Loader
 
-Please note, that Xdebug and IonCube Loader has some issues. With PHP 5.x, both extensions won't work at the same time, as long as encrypted plugins are activated. The debugger won't stop at your breakpoints and/or the whole request may crash. Disabling all encrypted plugins will solve the problem.
+Please note, that Xdebug and IonCube Loader has some issues.
+With PHP 5.x, both extensions won't work at the same time, as long as encrypted plugins are activated.
+The debugger won't stop at your breakpoints and/or the whole request may crash. Disabling all encrypted plugins will solve the problem.
 
-However, since PHP 7.0 the compatibility has improved. You're able to use Xdebug, IonCube Loader and encrypted plugins all together. Xdebug will stop at your breakpoints and everything is fine. You're not able to debug encrypted source code, of course. All encrypted code will just be skipped.
+However, since PHP 7.0 the compatibility has improved.
+You're able to use Xdebug, IonCube Loader and encrypted plugins all together.
+Xdebug will stop at your breakpoints and everything is fine.
+You're not able to debug encrypted source code, of course.
+All encrypted code will just be skipped.
 
-It's imported that the IonCube Loader extension is loaded BEFORE the Xdebug extension in your `php.ini`. For more troubleshooting consider the [PHPStorm manual](https://confluence.jetbrains.com/display/PhpStorm/Troubleshooting+PhpStorm+debugging#TroubleshootingPhpStormdebugging-DebuggingwithionCubeinstalled).
+It's imported that the IonCube Loader extension is loaded BEFORE the Xdebug extension in your `php.ini`.
+For more troubleshooting consider the [PHPStorm manual](https://www.jetbrains.com/help/phpstorm/troubleshooting-php-debugging.html#debugging-with-ioncube).
 
```

Comment:
The lines regarding ioncube compatibility can be removed completely I think, since it's not in use anymore.